apps/me/Dockerfile

```
@@ -0,0 +1,58 @@
+# ---------- Stage 1: Prune monorepo ----------
+FROM node:20-alpine AS builder
+
+ENV TURBO_TELEMETRY_DISABLED=1
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN apk add --no-cache libc6-compat && apk update
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)